### PR TITLE
feat: plus buttons

### PR DESCRIPTION
**Pull request by omBratteng**
## Changes

Depends on https://github.com/dailydotdev/daily-api/pull/2430

## Events

Did you introduce any new tracking events?

## Experiment

Did you introduce any new experiments?

## Manual Testing

> [!CAUTION]
> Please make sure existing components are not breaking/affected by this PR

AS-700

### Preview domain
https://as-700.preview.app.daily.dev

### CODE REVIEW

File: packages/shared/src/svg/LogoIcon.tsx [hunk 1]
```diff
@@ -20,10 +20,10 @@ export default function LogoIcon({ className }: LogoIconProps): ReactElement {
         className={className?.group}
       >
         <path
-          d="M26.633 8.69l-3.424-3.431 1.711-3.43 5.563 5.575c.709.71.709 1.861 0 2.572l-6.847 6.86c-.709.711-1.858.711-2.567 0a1.821 1.821 0 010-2.571l5.564-5.575z"
+          d="M29.5925 9.99823L25.7884 6.1862L27.6895 2.37549L33.8703 8.5693C34.6579 9.35848 34.6579 10.638 33.8703 11.4272L26.2629 19.0506C25.4753 19.8398 24.1985 19.8398 23.411 19.0506C22.6234 18.2614 22.6234 16.9819 23.411 16.1927L29.5925 9.99823Z"
```
Comment (rebelchris) [inline packages/shared/src/svg/LogoIcon.tsx:23]: What are the changes here?

**Review by rebelchris — APPROVED** [review]
One generic I see I forgot to mention is that the whole plus experience is feature flagged.
Can you maybe hook it up to feature?

(Can be separate PR though)

File: packages/shared/src/components/profile/ProfileSettingsMenu.tsx [hunk 2]
```diff
@@ -54,6 +56,16 @@ const useMenuItems = (): NavItemProps[] => {
...
+        ...(!user?.isPlus
+          ? [
+              {
+                label: 'Upgrade to plus',
+                icon: <DevPlusIcon />,
+                href: plusUrl,
+                className: 'text-accent-bacon-default',
+              },
+            ]
+          : []),
```
Comment (rebelchris) [inline packages/shared/src/components/profile/ProfileSettingsMenu.tsx:68]: This one needs to toggle to become Manage Plus when you have it.
(Link to external, yet to be defined)

Screenshot 2024-11-13 at 10 17 22

**Review by rebelchris — CHANGES_REQUESTED** [review]

File: packages/shared/src/svg/LogoIcon.tsx (hunk 1 above)
Comment (omBratteng) [reply packages/shared/src/svg/LogoIcon.tsx:23]: No changes really, mostly viewport of svg changed, so the paths changed too.
I copied the SVG from figma

Comment (omBratteng) [reply packages/shared/src/svg/LogoIcon.tsx:23]: SCR-20241113-lwrn

This shows the difference

File: packages/shared/src/components/profile/ProfileSettingsMenu.tsx (hunk 2 above)
Comment (omBratteng) [reply packages/shared/src/components/profile/ProfileSettingsMenu.tsx]: Fixed

**Comment by omBratteng** [comment]
Will make it a feature in separate PR

**Review by rebelchris — APPROVED** [review]
This works for me :)

Also in my branch I introduced a context, maybe you can use that for the experiment?
(although hook might be easier)

**Pull request merged**